lib/drives/util/recurrence_options_edit.dart

```diff
@@ -233,10 +234,13 @@ class RecurrenceOptionsEditState extends State<RecurrenceOptionsEdit> {
   }
 
   Widget buildIndicator() {
-    return RecurrenceOptionsIndicator(
-      before: originalRecurrenceRule,
-      after: recurrenceOptions.recurrenceRule,
-      start: recurrenceOptions.startedAt.isAfter(DateTime.now()) ? recurrenceOptions.startedAt : DateTime.now(),
+    return ExpandableSection(
```

Comment:
This still needs to set the parameter to decide whether it is expanded by default

Reply:
Yes I know. Working on that tomorrow